plugins/katello/3.11/installation/index.md

```diff
@@ -3,7 +3,7 @@ layout: plugins/katello/documentation
 title: Katello Installation
 version: 3.11
```

Comment:
How about:

```suggestion
version: '3.11'
```

The benefit would be that once you reach 3.20 you won't need to change it then. Otherwise it'll be interpreted as 3.2.

Reply:
You got me there. Updated!